windowing.py

```diff
@@ -91,39 +90,38 @@ def refine_start_time(windowed_data, threshold):
     index_of_start = indexes[0]
     return windows[index_of_start][0][3][0]
 
-# Input: lift_windows = np array
-# Input: rows = int (# of rows within a trace (ex: x, y, z, nanos => 4 rows))
-# Input: window_size = int (# of samples within one window when creating the windows)
-# Input: stride = int (# of smaples in between the start of each window)
-# Input: variane_threshold = int (variance threshold that determines
-#								  wether the current window is part of the lift)
-#
-# Description: Finds the more precise start time by creating much smaller windows
-#			  within the first window found to contain part of the lift ouputted
-#			  from initial_find_lift.
-#
-# Return: start_time = int (more precise start time of the lift in nanoseconds)
-
 
 def find_precise_start_time(lift_windows, rows, window_size, stride, variance_threshold):
+    # Description: Finds the more precise start time by creating much smaller windows
+    #			  within the first window found to contain part of the lift ouputted
+    #			  from initial_find_lift.
+    #
+    # Input: lift_windows = np array
+    # Input: rows = int (# of rows within a trace (ex: x, y, z, nanos => 4 rows))
+    # Input: window_size = int (# of samples within one window when creating the windows)
+    # Input: stride = int (# of smaples in between the start of each window)
+    # Input: variane_threshold = int (variance threshold that determines
+    #								  wether the current window is part of the lift)
+    #
+    #
+    # Return: start_time = int (more precise start time of the lift in nanoseconds)
     front = lift_windows[0]
     np_front = np.array(front)
     windowed_data = view_as_windows(np_front, (rows, window_size), step=stride)
     start_time = refine_start_time(windowed_data, variance_threshold)
```

Comment:
is it necessary to have the refine_start_time function or could you use the same function with a different parametrization and then just select the start instead of returning start & end?

or make it return only start or start& end with a flag?